pkg/apis/provisioning/v0alpha1/types.go

```diff
@@ -295,6 +296,43 @@ type FileItem struct {
 	Author   string `json:"author,omitempty"`
 }
 
+// Information we can get just from the file listing
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+type ObjectList struct {
```

Comment:
```suggestion
type ResourceList struct {
```

Isn't this the list of resources in the repository, shouldn't we call it as such (ResourceList, ResourceListItem, etc)?